membrane_rtc_engine/lib/membrane_rtc_engine/engine.ex

```diff
@@ -317,6 +317,14 @@ defmodule Membrane.RTC.Engine do
     Pipeline.call(rtc_engine, :get_endpoints)
   end
 
+  @doc """
+  Returns number of forwarded tracks in RTC Engine.
+  """
+  @spec get_forwarded_tracks_number(rtc_engine :: pid()) :: integer()
+  def get_forwarded_tracks_number(rtc_engine) do
+    Pipeline.call(rtc_engine, :get_forwarded_tracks_number)
```

Comment:
+1'ing the suggestion of @roznawsk -- I think `get_forwarded_track(s)_count`, `get_forwarded_track(s)_amount` or `get_num_forwarded_tracks` would be more descriptive